cluster-operator/src/test/java/io/strimzi/operator/cluster/operator/resource/cruisecontrol/MockCruiseControl.java

```diff
@@ -514,6 +514,27 @@ public void setupCCUserTasksCompletedWithError() throws IOException, URISyntaxEx
                                 .withDelay(TimeUnit.SECONDS, 0));
     }
 
+    public void setupUserTasktoEmpty() throws IOException, URISyntaxException {
```

Comment:
Who is throwing these exceptions? I guess we don't need them.

Reply:
Removed